test/features/steps/steps.py

```diff
@@ -336,6 +336,29 @@ def reload_page_every_n_until_find(context, xpath, seconds=5, reload_times=5):
     assert False, 'Element with xpath "{}" was not found'.format(xpath)
 
 
+@step(u'I trigger notification about updated privacy assessment results')
+def i_tigger_notification_assessment_results(context):
```

Comment:
Misspelled "trigger"

Reply:
Will be fixed and pushed by Alex shortly.